locales/de.json

```diff
@@ -0,0 +1,96 @@
+{
+	"File": "Datei",
+	"Quit": "Schliessen",
+	"Edit": "Bearbeiten",
+	"Undo": "Lösen",
+	"Redo": "Noch einmal",
+	"Cut": "Schneiden",
+	"Copy": "Kopieren",
+	"Paste": "Einfügen",
+	"Delete": "Löschen",
+	"Select all": "Alles markieren //?",
+	"View": "Wiedergabe",
```

Comment:
This should be either `Ansicht` or `Wiedergabe`.
Ansicht is correct translation if the menu entry is meant which allows you to configure the windows etc. Wiedergabe is good to use if you mean something like "view the video".

Reply:
This is a menu entry, so Ansicht should be used. Thanks.